templates/partial/_tabs-navigation.html

```diff
@@ -10,7 +10,7 @@
           <a href="#configuration" class="p-tabs__link" tabindex="-1" role="tab"
             aria-controls="configuration">Configuration</a>
         </li>
-        {% if entity_type == "charm" %}
+        {% if package["package-type"] == "charm" %}
```

Comment:
does this not work as `package.package-type` to keep it consistent?

Reply:
"-" is a reserved character, we would need to change all "-"'s to "_"'s in the backend, which might be worth doing at some point.